kongswap_adaptor/src/deposit.rs

```diff
@@ -376,10 +381,10 @@ impl<A: AbstractAgent> KongSwapAdaptor<A> {
 }
 
 #[cfg(test)]
-mod deposit_add_pool;
+mod test_failed_transfer_from;
 
 #[cfg(test)]
-mod test_failed_transfer_from;
+mod deposit_add_liquidity;
```

Comment:
Choose a naming convention and follow it. 

```suggestion
mod test_deposit_add_liquidity;
```

Reply:
Will be solved in the upcoming PRs.